docs/administrator/monitoring/working-with-prometheus-in-control-plane.md

```diff
@@ -0,0 +1,284 @@
+         - role: node
+         relabel_configs:
+         - target_label: __address__
+           replacement: kubernetes.default.svc:443
+         - source_labels: [__meta_kubernetes_node_name]
+           regex: (.+)
+           target_label: __metrics_path__
+           replacement: /api/v1/nodes/${1}/proxy/metrics/cadvisor
+         - action: labelmap
+           regex: __meta_kubernetes_node_label_(.+)
+         metric_relabel_configs:
+         - action: replace
+           source_labels: [id]
+           regex: '^/machine\.slice/machine-rkt\\x2d([^\\]+)\\.+/([^/]+)\.service$'
+           target_label: rkt_container_name
+           replacement: '${2}-${1}'
+         - action: replace
+           source_labels: [id]
+           regex: '^/system\.slice/(.+)\.service$'
+           target_label: systemd_service_name
+           replacement: '${1}'
+         - source_labels: [pod]
+           separator: ;
+           regex: (.+)
+           target_label: pod_name
+           replacement: $1
+           action: replace
+   ---
+   apiVersion: v1
+   kind: "Service"
+   metadata:
+     name: prometheus
+     namespace: monitor
+     labels:
+       name: prometheus
+   spec:
+     ports:
+     - name: prometheus
+       protocol: TCP
+       port: 9090
+       targetPort: 9090
+       nodePort: 31801 
+     selector:
+       app: prometheus
+     type: NodePort
+   ---
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     labels:
+       name: prometheus
+     name: prometheus
+     namespace: monitor
+   spec:
+     replicas: 1
+     selector:
+       matchLabels:
+         app: prometheus
+     template:
+       metadata:
+         labels:
+           app: prometheus
+       spec:
+         serviceAccountName: prometheus
+         containers:
+         - name: prometheus
+           image: prom/prometheus:latest
+           command:
+           - "/bin/prometheus"
+           args:
+           - "--config.file=/etc/prometheus/prometheus.yml"
+           - "--storage.tsdb.path=/prom-data"
+           - "--storage.tsdb.retention.time=180d"
+           ports:
+           - containerPort: 9090
+             protocol: TCP
+           volumeMounts:
+           - mountPath: "/etc/prometheus"
+             name: prometheus-config
+           - mountPath: "/prom-data"
+             name: prom-data
+         initContainers:
+         - name: prometheus-data-permission-fix
+           image: busybox
+           command: ["/bin/chmod","-R","777", "/data"]
+           volumeMounts:
+           - name: prom-data
+             mountPath: /data
+         volumes:
+         - name: prometheus-config
+           configMap:
+             name: prometheus-config
+         - name: prom-data
+           hostPath:
+             path: /var/lib/prom-data
+             type: DirectoryOrCreate
+   
+   ```
+
+3. Use any node IP of the control plane and the port number (default 31801) to enter the Prometheus monitoring page of the control plane
+
```

Comment:
BTW，Can you describe how to access Grafana and show the Grafana control panel？

Reply:
done